.deepsource.toml

```diff
@@ -1,19 +1,22 @@
 version = 1
 
-test_patterns = ["*/test/**"]
+test_patterns = [
+    "**/*_test.go",
+    "tests/**"
+]
 
 [[analyzers]]
 name = "go"
 enabled = true
 
   [analyzers.meta]
-  import_paths = ["github.com/rudderlabs/rudder-server"]
-  dependencies_vendored = "false"
+  import_paths = ["github.com/sourya-deepsource/rudder-server"]
```

Comment:
@sourya-deepsource Shouldn't this point to rudderlabs itself?

Reply:
Oops! My bad. Yes